cmd/interceptors/main.go

```diff
@@ -19,25 +19,36 @@ package main
 import (
 	"context"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"net"
 	"net/http"
+	"os"
 	"time"
 
 	"github.com/tektoncd/triggers/pkg/interceptors/server"
 	"go.uber.org/zap"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubeclientset "k8s.io/client-go/kubernetes"
 	secretInformer "knative.dev/pkg/client/injection/kube/informers/core/v1/secret"
 	"knative.dev/pkg/injection"
 	"knative.dev/pkg/logging"
 	"knative.dev/pkg/signals"
+	certresources "knative.dev/pkg/webhook/certificates/resources"
 )
 
 const (
-	// Port is the port that the port that interceptor service listens on
-	Port         = 8082
+	// HTTPSPort is the port where interceptor service listens on
+	HTTPSPort    = 8443
```

Comment:
question: is it necessary to change the port?

Reply:
In order to make it sync with `https` as `https` usually run on `8443` or `443` so just modified